apb-base/Dockerfile-canary

```diff
@@ -41,7 +53,12 @@ RUN echo "localhost ansible_connection=local" > /etc/ansible/hosts \
 COPY files/etc/ansible/* /etc/ansible/
 COPY files/usr/bin/* /usr/bin/
 
-RUN useradd -u 1001 -r -g 0 -M -b /opt/apb -s /sbin/nologin -c "apb user" apb
-RUN chown -R apb: /opt/{ansible,apb}
+ RUN mkdir -p /usr/share/ansible/openshift \
+              /etc/ansible /opt/ansible \
+              ${BASE_DIR}/{etc,.kube,.ansible/tmp} \
+              && useradd -u ${USER_UID} -r -g 0 -M -d ${BASE_DIR} -b ${BASE_DIR} -s /sbin/nologin -c "apb user" ${USER_NAME} \
```

Comment:
this shouldn't be necessary... the etc/passwd entry is created automatically at runtime. take a look...
https://github.com/fusor/apb-examples/blob/1ab6a904743390db003faabb2b669703821a3626/apb-base/files/usr/bin/entrypoint.sh#L29-L31

Reply:
nm, removing my review... i see now this was pre-existing code? i need to improve w/ separate PR.